primefaces/src/main/java/org/primefaces/component/tabview/TabViewRenderer.java

```diff
@@ -316,7 +315,8 @@ protected void encodeContents(FacesContext context, TabView tabView) throws IOEx
 
         tabView.forEachTab((tab, i, active) -> {
             try {
-                encodeTabContent(context, tab, i, active, dynamic, repeating);
+                String tabindex = active ? tabView.getTabindex() : "-1";
```

Comment:
Hi, I had a question about this function, if it is 'active`, does `getTabIndex()` return `"0"`?

Reply:
Gotcha! thank you!